Docs/LinuxOsxInstall.md

```diff
@@ -13,15 +13,21 @@
 - Make sure you have OpenGL availible
 
 ## Linux installation instructions
+### Arch 
+The AUR package [`factorio-yafc-git`](https://aur.archlinux.org/packages/factorio-yafc-git) points to the original upstream project by ShadowTheAge. Clone the AUR package and edit the url to the following: https://github.com/have-fun-was-taken/yafc-ce
```

Comment:
```suggestion
The AUR package [`factorio-yafc-git`](https://aur.archlinux.org/packages/factorio-yafc-git) points to the original upstream project by ShadowTheAge. To make this package suitable for YAFC-CE take the following steps:
1. Clone the AUR package
1. Open PKGBUILD
3. Edit the `source` to use the following URL: https://github.com/have-fun-was-taken/yafc-ce
```

Sorry I was too quick, the instructions didn't make sense anymore :disappointed: